mantidproject/mantid
# Remove as many `QT_VERSION_CHECK`'s as possible

**Description of work.**
This PR removes as many `QT_VERSION_CHECK`'s for qt4 as possible.

**To test:**
1. Code review
2. grep for `QT_VERSION_CHECK` and check that all that can be removed, has been removed
3. Ensure all Qt4 code in pqHelpWindow.h and pqHelpWindow.cxx is removed.

Part of #30268

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
